odoo/fields.py

```diff
@@ -2016,6 +2017,22 @@ def _convert(self, value, record, validate):
                     # sanitized. It means that someone who was part of a group
                     # allowing to bypass the sanitation saved that field
                     # previously.
+
+                    diff = unified_diff(
+                        original_value_sanitized.splitlines(),
+                        original_value_normalized.splitlines(),
+                    )
```

Comment:
```suggestion
                    diff = unified_diff(
                        original_value_normalized.splitlines(),
                        original_value_sanitized.splitlines(),
                    )
```
I would have probably picked this way so we see what would be removed by the sanitize and prevents the edition of the field
![image](https://github.com/odoo/odoo/assets/30048408/5737a88b-b956-4d04-9dc4-77713b730a6a)

Instead of 
![image](https://github.com/odoo/odoo/assets/30048408/c5b57519-c436-4095-a9aa-469f4731b37e)


But it's just a matter of taste so it's not worth debating